testsuite/cluster-test/cluster_test_pod_sample.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
How would we use this file? If you want to put it into repo, it probably should not contain smth like `kush_bbe30c38`

Reply:
Removed file